sources/Inputs/include/HvdcLine.h

```diff
@@ -0,0 +1,77 @@
+//
+// Copyright (c) 2020, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+
+/**
+ * @file  HvdcLine.h
+ *
+ * @brief HvdcLine structure header file
+ *
+ */
+
+#pragma once
+
+#include "Behaviours.h"
+
+#include <boost/optional.hpp>
+#include <string>
+
+namespace dfl {
+namespace inputs {
+
+/**
+ * @brief Hvdc Line structure
+ */
+struct HvdcLine {
+  using HvdcLineId = std::string;   ///< HvdcLine id definition
+  using ConverterId = std::string;  ///< alias for converter id
+  using BusId = std::string;        ///< alias for bus id
+
+  /// @brief Type of converter
+  enum class ConverterType {
+    VSC,  ///< voltage source converter
+    LCC   ///< line-commutated converter
+  };
+
+  /**
+   * @brief Determines if two converter interfaces are equal
```

Comment:
converters'

Reply:
For me we can keep "converter interfaces" as it is closer to the type name

Reply:
ok